dtfs-central-api/src/v1/controllers/utilisation-report-service/post-remove-fees-from-payment-group.controller/index.ts

```diff
@@ -39,9 +42,13 @@ export const postRemoveFeesFromPayment = async (req: PostRemoveFeesFromPaymentRe
 
     const allFeeRecords = payment.feeRecords;
 
-    validateNotAllFeeRecordsSelected(selectedFeeRecordIds, allFeeRecords.length);
+    validateSelectedFeeRecordsExistInPayment(selectedFeeRecordIds, allFeeRecords);
+
+    const totalFeeRecordsOnPayment = allFeeRecords.length;
+    validateSelectedFeeRecordsDoesNotExceedTotal(selectedFeeRecordIds, totalFeeRecordsOnPayment);
+    validateNotAllFeeRecordsSelected(selectedFeeRecordIds, totalFeeRecordsOnPayment);
```

Comment:
After thinking about it I agree this validation does add up but I think can we can simplify/consolidate it so it reads a little easier, let's chat offline

Reply:
Looks a lot nicer now thanks - changes in e4891b3fe9594a1b9b11d18f50a51d16502867ad